.github/pull_request_template.md

```diff
@@ -36,7 +36,7 @@ Related changes
 If no, just leave the release note block below as is.
 If yes, a release note is required:
 Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
-Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/docs/release-note-guide.md) before writing the release note.
+Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.
```

Comment:
Can you open a separate PR for this? We can merge this first and don't mix two things together.

Reply:
> Can you open a separate PR for this? We can merge this first and don't mix two things together.

I have created #1600 for this.